app/controllers/course/virtual_classrooms_controller.rb

```diff
@@ -16,6 +16,30 @@ def access_link #:nodoc:
     end
   end
 
+  def recorded_videos #:nodoc:
+    authorize! :manage, @virtual_classroom
+    @braincert_api_service = Course::VirtualClassroom::BraincertApiService.new(
+      @virtual_classroom, @settings
+    )
+    respond_to do |format|
+      format.json do
+        render json: @braincert_api_service.fetch_recorded_videos
+      end
+    end
+  end
+
+  def recorded_video_link #:nodoc:
+    authorize! :manage, Course::VirtualClassroom
```

Comment:
OK, I did some tests, from my tests, even students can access this, the following test will pass:
```
    subject { Ability.new(user) }
    context 'when the user is a Course Student' do
      let(:user) { create(:course_student, course: course).user }

      it { is_expected.to be_able_to(:manage, Course::VirtualClassroom) }
    end
```

Maybe you can do a controller test as a student to verify ?